contracts/PartyBuy.sol

```diff
@@ -66,21 +71,24 @@ contract PartyBuy is Party {
 
     function initialize(
         address _nftContract,
-        uint256 _tokenId,
         uint256 _maxPrice,
         uint256 _secondsToTimeout,
+        address[] calldata _deciders,
         Structs.AddressAndAmount calldata _split,
         Structs.AddressAndAmount calldata _tokenGate,
         string memory _name,
         string memory _symbol
     ) external initializer {
-        // validate maxPrice
-        require(_maxPrice > 0, "PartyBuy::initialize: must set price higher than 0");
         // initialize & validate shared Party variables
-        __Party_init(_nftContract, _tokenId, _split, _tokenGate, _name, _symbol);
+        __Party_init(_nftContract, _split, _tokenGate, _name, _symbol);
         // set PartyBuy-specific state variables
         expiresAt = block.timestamp + _secondsToTimeout;
         maxPrice = _maxPrice;
+        require(_maxPrice + _getEthFee(_maxPrice) >= _maxPrice, "PartyBuy::initialize: Max price can overflow");
+        // set deciders list
+        for (uint256 i = 0; i < _deciders.length; i++) {
```

Comment:
Do we want to require that there is at least 1 decider?  Otherwise the party would get stuck right?

Reply:
great suggestion 😀  https://github.com/PartyDAO/partybuy/pull/4/commits/af12ac0ead81f606ece5a4992cca1d3fa2ff8635